include/srf/types.hpp

```diff
@@ -41,18 +41,15 @@ using Mutex = userspace_threads::mutex;  // NOLINT(readability-identifier-naming
 
 using CondV = userspace_threads::cv;  // NOLINT(readability-identifier-naming)
 
-using InstanceID = std::uint32_t;                      // NOLINT(readability-identifier-naming)
-using MachineID  = std::uint32_t;                      // NOLINT(readability-identifier-naming)
-using Instances  = std::map<InstanceID, std::string>;  // NOLINT(readability-identifier-naming)
+using InstanceID = std::uint64_t;                      // NOLINT(readability-identifier-naming)
```

Comment:
I added TagID here which helped distinguish it from InstanceID.

Reply:
Done